### ISSUE TITLE
Update Naming Order for faces-3.0 facesProvider

### ISSUE THREAD

**Issue by volosied** [OpenLiberty/open-liberty#14436]
Request made in the externals review to have myfaces before the version. 

io.openliberty.facesProvider-3.0.0.MyFaces  --> io.openliberty.facesProvider-MyFaces.3.0.0 

https://github.com/OpenLiberty/open-liberty/blob/a7694036f901962827d5e82a95fd4c3ae2c6c58c/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.facesProvider-3.0.0.MyFaces.feature

**Comment by volosied** [comment]
After testing with my current facesContainer-3.0 feature and the faces-3.0,  the singleton code does not work  as  expected.

The  _CWWKF0033E: The singleton features io.openliberty.facesProvider-3.0.0.Container and io.openliberty.facesProvider-3.0.0.MyFaces cannot be loaded at the same time._  message does not occur with the new name change described above.  

Here is a list of  changes when I tested:

```
visibility/public/facesContainer-3.0/io.openliberty.facesContainer-3.0.feature
20:  io.openliberty.facesProvider-3.0.0.Container, \

visibility/public/faces-3.0/io.openliberty.faces-3.0.feature
44: io.openliberty.facesProvider-Container.3.0.0, \

visibility/private/io.openliberty.facesProvider-MyFaces.3.0.0.feature
4:symbolicName=io.openliberty.facesProvider-MyFaces.3.0.0

visibility/private/io.openliberty.facesProvider-Container.3.0.0.feature
4:symbolicName=io.openliberty.facesProvider-Container.3.0.0
```

**Comment by tjwatson** [comment]
I remember discussing this during the externals but don't recall the conclusion.  I only recall that we were confused by the use of qualifier `MyFaces` for the feature name, but at the time we did have the complete picture with the `Container` variant of the private feature.

During the call I did confirm that having a `qualifier` in the version portion does work at runtime and indeed it would make the following singleton features conflict with each other:

- io.openliberty.facesProvider-3.0.0.MyFaces
- io.openliberty.facesProvider-3.0.0.Container

This is a reasonable use of a `qualifier` for private feature in order to cause a conflict between the two public features `faces-3.0` and (yet to be introduced) `facesContainer-3.0`.
